.github/checkgroup.yml

```diff
@@ -73,6 +73,13 @@ subprojects:
   - id: "pytorch_lightning: Docker"
     paths:
       - "dockers/**"
+      - "!dockers/README.md"
```

Comment:
@Borda where did you see that negation works?

Reply:
it was in GitHub docs

Reply:
Maybe not, my bad...